content/go/concepts/Variables/Variables.md

```diff
@@ -0,0 +1,126 @@
+
+---
+Title: 'Variables'
+Description: 'Variables are used to store numbers, strings, true/false boolean states, or objects based on their type when declaring them'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Variables'
+  - 'Booleans'
+  - 'Strings'
+  - 'Integers'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+---
+
+## Definition
+
+Variables are used to store user information like numbers, strings or true/false state to be used in the program later on.
+
+## Difference from constants 
+
+Unlike constants, when declaring a variable, its value can be reassigned later on but with restriction to be similar to the data type when initialized.
+
+### Example using pseudo
+
+This is acceptable
+
+```pseudo
+declaring an int variable 
+use it in a while loop
+reassign it to zero after
+```
+
+This will result in an error
+
+```pseudo
+declaring an int variable 
+use it in a while loop
+reassign it to "Codecaedmy" after
+```
+
+## syntax explination
+
+```Go
+var 'variable name' 'var type'
+```
+
+first begin with the `var` keyword followed by the name of the variable, and at the end the variable type, each of these are seperated by spaces.
+
+### variable names should:
+
+- begin with a letter 
+- must not contain special characters (only letters and digits)
+- must not be a GO keyword 
+- names are case sensitive
+ 
+### some of the basic types are:
+
+- int
```

Comment:
This line and the following two should be in backticks as they are Go keywords.

Example: `int`.